### PR TITLE
[GEOS-9126] Document how to export styles from QGIS

### PR DESCRIPTION
**Pull request by aaime**
The vector part can be performed using existing QGIS releases, the raster portion requires a recently merged PR, see https://github.com/qgis/QGIS/pull/9049 (merged yesterday)

### CODE REVIEW

File: doc/en/user/source/styling/qgis/index.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+   supported in SLD, but QGIS won't export it). If you are interested, both projects would welcome 
+   sponsoring to improve the situation.
+
+
+Exporting vector symbology
+--------------------------
+
+This is a step by step guide to style a GeoServer demo layer, "sfdem".
+
+- Open QGIS (minimum version 3.0)
```
Comment (jodygarnett) [inline doc/en/user/source/styling/qgis/index.rst]: Numbered steps help

**Comment by jodygarnett** [comment]
@aaime I have pushed up a small change to save time, provides numbered steps and makes use of additional sphinx / rst markup.

**Comment by aaime** [comment]
Thank you. Just one question, how did you do that? Thought you had to make a PR against my branch, but this commit apparently went straight into the PR.

**Comment by jodygarnett** [comment]
When you made your PR you invited repository owners to collaborate with you.

I added you as a remote, checked out the branch locally with tracking, and revised the content, pushing up to your branch when done.

**Pull request merged**